mods/_various/simple_chat_dice/mod.conf

```
@@ -0,0 +1,3 @@
+name = dice
+description = Minetest Game mod: dice
```

это не из MTG.
тут должно быть что-то вроде:
```suggestion
description = Adds `/dice` chat command for printing random numbers
```